Gemfile.lock

```diff
@@ -25,6 +25,7 @@ GEM
     ethon (0.15.0)
       ffi (>= 1.15.0)
     eventmachine (1.2.7)
+    eventmachine (1.2.7-x64-mingw32)
```

Comment:
Please remove these changes, thanks!

Reply:
@MikeMcQuaid  All of them or just 28th line ```eventmachine (1.2.7-x64-mingw32) ```

Reply:
All `Gemfile` related changes, thanks!

Reply:
Thanks for assisting!
 it shows same error when I type: ```script/test```